maas-api/internal/models/kserve_llmisvc_test.go

```diff
@@ -293,6 +293,7 @@ func TestListAvailableLLMs(t *testing.T) { //nolint:maintidx // linter is compla
 				&kservefakev1beta1.FakeServingV1beta1{Fake: fakeKServe},
 				&kservefakev1alpha1.FakeServingV1alpha1{Fake: fakeKServe},
 				&gatewayfake.FakeGatewayV1{Fake: fakeGateway},
+				nil, // k8sClient not needed for this test
```

Comment:
I wonder if we want to provide some tests that actually demonstrate that the behaviour works as intended.